models/test_suite/reachability_analysis/site_accross_bonds_domain_3/site_accross_bonds_domain.ka

```diff
@@ -1,13 +1,13 @@
-%agent:A(z,t)
-%agent:B(z,t)
-%agent:E(z,t)
-%agent:F(z,t)
-%agent:C(x)
-%agent:D(x)
-%agent:G(x)
-%agent:H(x)
--> C(x!1),A(z!1,t!2),B(t!2,z!3),C(x!3) @1
-C(x!1),A(z!1,t!2),B(t!2,z!3),C(x!3) -> D(x!1),A(z!1,t!2),B(t!2,z!3),D(x!3) @1
-C() -> @ 1
--> C(x!1),E(z!1,t!2),F(t!2,z!3),D(x!3) @1
-C(x!1),E(z!1,t!2),F(t!2,z!3),D(x!3) -> G(x!1),E(z!1,t!2),F(t!2,z!3),H(x!3) @1
+%agent: A(z,t)
+%agent: B(z,t)
+%agent: E(z,t)
+%agent: F(z,t)
+%agent: C(x)
+%agent: D(x)
+%agent: G(x)
+%agent: H(x)
+.,.,.,. -> C(x[1]),A(z[1],t[2]),B(t[2],z[3]),C(x[3]) @ 1
+C(x[1]),A(z[1],t[2]),B(t[2],z[3]),C(x[3]),.,.,.,. -> .,.,.,.,D(x[1]),A(z[1],t[2]),B(t[2],z[3]),D(x[3]) @ 1
```

Comment:
Again: accurate translation but I worry that the test meant to be `.,C(x[1]),A(z[1],t[2]),B(t[2],z[3]),C(x[3]),. -> D(x[1]),.,A(z[1],t[2]),B(t[2],z[3]),.,D(x[3]) @ 1` and wonder whether we should change it. What do you think, @feret ?

Reply:
Same thing here but more sensitive. 
The point is to test agent creation in the abstract domain "site_accross_domain".